build.sbt

```diff
@@ -36,7 +36,7 @@ val sharedSettings = Seq(
            "-Ywarn-value-discard"
          )
        else if (scalaBinaryVersion.value.startsWith("2.13"))
-         List("-Werror", "-Wdead-code", "-Wunused", "-Wvalue-discard")
+         List("-Werror", "-Wdead-code", "-Wunused", "-Wvalue-discard", "-Ytasty-reader")
```

Comment:
Oh nice! I didn't know. Does this mean the scala2 compiler can also read tasty files?

So I just searched a bit to answer myself :-) Found it [here](https://docs.scala-lang.org/scala3/guides/migration/compatibility-classpath.html#the-scala-213-tasty-reader). So scala2 can read tasty files, support is a bit limited of course, but this is really nice!

I tried later last night with setting the `crossScalaVersions` to an empty `Seq` in the root module only. This also seems to work :-). Not sure what's the better way - I'm wondering what's happening with the `-Ytasty-reader` here - does the akka module gets compiled against the calev-core-scala3 module? *If* that would be the case, I'm not sure if that is a good thing (shouldn't they be compiled against the core-scala2 variants?) I mean currently there are no scala3 features in the core module, and there won't be any in the near future, because I want to prioritize compatibility across scala versions. So I guess it doesn't matter. Wdyt?